install_windows_community

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env node
```

Comment:
as much as I like this script it has nothing to do with the slider ;)

@Pr0chin please remove this file from this PR